.gitignore

```diff
@@ -2,31 +2,31 @@
 .DS_Store
 
 # directory for developers to store local resources
-/.local/
+.local/
 
-/*.code-workspace
+*.code-workspace
 
 node_modules
-/npm/*/bin
-/npm/workerd/install.js
-/npm/workerd/lib/
+npm/*/bin
+npm/workerd/install.js
+npm/workerd/lib/
 package-lock.json
 
 # The external link for compile_flags.txt: Differs on Windows vs macOS/Linux, so we can't check it in. The pattern needs to not have a trailing / because it's a symlink on macOS/Linux.
-/external
+external
```

Comment:
This changes the meaning to ignoring all folders named `external` instead of just ones in the root directory of the repo. Maybe zed should be changed to understand gitignore syntax?

Reply:
I can add "./" prefix which will limit the scope to the root folder.

Reply:
Yeah adding this to all of them would probably be a good idea.